compiler/circle-mpqsolver/src/Methods/BisectionSolver.cpp

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "BisectionSolver.h"
+#include "../utils/ErrorApproximator.h"
+#include "../utils/ModuleCloner.h"
+#include "../utils/Evaluator.h"
```

Comment:
It is not recommended to use `..` when including headers for maintainability. How about add below to `CMakeLists.txt`?
```cmake
target_include_directories(circle-mpqsolver PRIVATE src)
```
```suggestion
#include "utils/ErrorApproximator.h"
#include "utils/ModuleCloner.h"
#include "utils/Evaluator.h"
```